src/diffusers/loaders/unet.py

```diff
@@ -664,6 +664,44 @@ def delete_adapters(self, adapter_names: Union[List[str], str]):
             if hasattr(self, "peft_config"):
                 self.peft_config.pop(adapter_name, None)
 
+    def _convert_ip_adapter_to_diffusers(self, state_dict):
```

Comment:
```suggestion
    def _convert_ip_adapter_image_proj_to_diffusers(self, state_dict):
```

Reply:
I wonder if we should move this method to https://github.com/huggingface/diffusers/blob/main/src/diffusers/loaders/ip_adapter.py. This way the UNet loader stays cleaner.